test/e2e/network/proxy.go

```diff
@@ -487,24 +491,22 @@ func validateRedirectRequest(client *http.Client, redirectVerb string, urlString
 	framework.ExpectEqual(resp.StatusCode, 301, "The resp.StatusCode returned: %d", resp.StatusCode)
 }
 
-func checkPodStatus(f *framework.Framework, label string) func() (bool, error) {
-	return func() (bool, error) {
-		var err error
```

Comment:
if we are going to keep it I prefer to keep the same way it was, so an sporadic API call failure doesn't fail the test ... if the first call succeed wait.PollImmediate will have the same effect as this change

Reply:
I think that the net change is just adding 
```
			framework.ExpectNoError(e2epod.WaitForPodNameRunningInNamespace(f.ClientSet, "agnhost", f.Namespace.Name))
			framework.Logf("Pod is running now.")
```

to both tests and leaving the rest as is, sorry for the back and forth :smile: